recipes/opennavsurf-bag/bld.bat

```diff
@@ -0,0 +1,41 @@
+setlocal EnableDelayedExpansion
+
+mkdir build
+if errorlevel 1 exit 1
+
+if  %vc% LEQ 9 set MSVC_VER=1500
```

Comment:
I don't think these settings are needed. We have not been building for old VCs for a long time.

Reply:
Resolved.